src/new-client/src/components/styled.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components"
+
+export const HeaderAction = styled.button`
```

Comment:
This should have a more descriptive filename

Reply:
Fixed, it's `headerActionStyles` now